rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -24,11 +24,12 @@ BuildRequires: ruby(release)
 Requires: ruby(rubygems)
 Requires: foreman-proxy >= 1.8.0
 Requires: crontabs
+Requires: rubygem(openscap) >= 0.4.3
 BuildRequires: rubygems-devel
 BuildRequires: ruby
 BuildArch: noarch
 Provides: rubygem(%{gem_name}) = %{version}
-Obsoletes: rubygem-foreman-proxy_openscap <= 0.3.0-1
+Obsoletes: rubygem-foreman-proxy_openscap <= 0.4.1-1
```

Comment:
What about it?  I don't understand why you've changed this.  Was a newer version of foreman-proxy_openscap shipped?